examples/example-vite-react-app-recs/src/App.tsx

```
@@ -18,7 +19,11 @@ function App() {
 
     // sync the contract components to the local state
     // this fetches the entities from the world and updates the local state
-    useQuerySync(toriiClient, contractComponents as any, []);
+    useQuerySync(
+        toriiClient,
+        contractComponents as any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)